tests/test_config_entries.py

```diff
@@ -8039,18 +8042,9 @@ async def async_step_reconfigure(self, user_input=None):
 
             async def _async_step_confirm(self):
                 """Confirm input."""
-                try:
-                    entry = self._get_entry()
-                except ValueError as err:
-                    reason = str(err)
-                else:
-                    reason = f"Found entry {entry.title}"
-                try:
-                    entry_id = self._entry_id
-                except ValueError:
-                    reason = f"{reason}: -"
-                else:
-                    reason = f"{reason}: {entry_id}"
+                reason = f"Found entry {self._get_entry().title}"
+                reason = f"{reason},subentry_type={self._subentry_type}"
```

Comment:
Maybe it makes more sense to put the subentry type after the colon and the config entry id before the colon, since the rest of the reason before the colon refers to the config entry and the rest of the reason after the colon refers to the sub entry?